src/foam/nanos/dig/drivers/DigJsonDriver.js

```diff
@@ -176,13 +177,23 @@ foam.CLASS({
         formatter.setX(x);
 
         var p = x.get(HttpParameters.class);
-        if ( p != null && "false".equals(p.getParameter("multiline")) ) {
-          formatter.setMultiLine(false);
-        } else {
-          formatter.setMultiLine(true);
+        if ( p != null ) {
+          if ( isSetToDisabled(p.getParameter("multiline")) ) formatter.setMultiLine(false);
+          if ( isSetToDisabled(p.getParameter("shortname")) ) formatter.setOutputShortNames(false);
         }
         return formatter;
       `
+    },
+    {
+      name: 'isSetToDisabled',
```

Comment:
Logic seems reversed here. It would be more normal to sai isEnabled().

Reply:
we wanted to capture the fact that it's default to enabled when not set (not explicitly set to false).